app/views/content/complaints.html.erb

```diff
@@ -8,24 +8,20 @@
       <li>make a complaint</li>
       <li>give feedback</li>
       <li>get help with a technical problem</li>
-      <li>let us know if a teacher training provider has treated you unfairly - for example, you’ve experienced discrimination or bias</li>
+      <li>let us know if you've been treated unfairly - for example, you’ve experienced discrimination or bias from a teacher training provider</li>
     </ul>
 
     <p class="govuk-body">We’ll respond within 5 to 7 working days, or one working day if a technical problem prevents you from progressing your application.</p>
 
-    <h2 class="govuk-heading-m">If you need advice about your teacher training application</h2>
-
-    <p class="govuk-body"><%= govuk_link_to 'Call us or chat online', t('get_into_teaching.url_online_chat') %> instead.</p>
-
     <h2 class="govuk-heading-m">If you let us know about discrimination, bias or unfairness</h2>
 
-    <p class="govuk-body">We’ll email you with some advice about where to get support.</p>
+    <p class="govuk-body">We'll ask you whether you'd like us to share your experience with the teacher training provider, so that we can ask the provider to take action.</p>
```

Comment:
Minor. (Switch to curly apostrophes):

```suggestion
    <p class="govuk-body">We’ll ask you whether you’d like us to share your experience with the teacher training provider, so that we can ask the provider to take action.</p>
```